README.md

```diff
@@ -5,8 +5,15 @@ This project replaces the legacy "VIVO Installer". It is used to create a "third
 ## Usage
 
 1. Clone this repository
-1. Add any overlay files to their expected location with the `src` directory tree
-1. Build the custom webapp
+2. Add any overlay files to their expected location with the `src` directory tree
```

Comment:
In case you were unaware, GitHub markdown increments the numbers in an ordered list if they all are set as `1.`... but manually setting them as you have done here also works.

Reply:
I did not know that. I just assumed it was a mistake. I should of known better. ;)